packages/web-app-files/tests/unit/components/SideBar/Shares/FileShares.spec.ts

```diff
@@ -79,20 +73,16 @@ describe('FileShares', () => {
       expect(wrapper.html()).toMatchSnapshot()
     })
     it('reacts on delete events', async () => {
-      const spyOnCollaboratorDeleteTrigger = vi
-        .spyOn(FileShares.methods, '$_ocCollaborators_deleteShare_trigger')
-        .mockImplementation(() => undefined)
       const { wrapper } = getWrapper({ collaborators })
-      wrapper
-        .findComponent<typeof CollaboratorListItem>('collaborator-list-item-stub')
-        .vm.$emit('onDelete')
+      const { dispatchModal } = useModals()
+      ;(wrapper.findComponent<any>('collaborator-list-item-stub').vm as any).$emit('onDelete')
```

Comment:
meh, can't we use proper types with stubs? :-\

Reply:
https://github.com/owncloud/web/pull/10677